src/projetos/CadastroProjetosPage.tsx

```
@@ -0,0 +1,395 @@
+        message: "Deve haver pelo menos 1 vaga.",
+    }),
+    horas: z.coerce.number().min(1, {
+        message: "Projeto deve ter pelo menos 1 hora.",
+    }),
+    professor: z.string().min(1, {
+        message: "Selecione um professor.",
+    }),
+})
+
+export function CadastroProjetoPage() {
+    const [departamentos, setDepartamentos] = useState<Departamentos[]>([])
```

ESLint

[error] Delete `··`

(prettier/prettier)